### PR TITLE
[ST-732] Maven plugin

### PR DESCRIPTION
**Pull request by Konboi**
SSIA

I wrote the document in another PR https://github.com/launchableinc/cli/pull/119, so please see it if you know the useage.

### CODE REVIEW

File: launchable/test_runners/maven.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import os
+import click
+from . import launchable
+
+
+@click.argument('source_roots', required=True, nargs=-1)
+@launchable.subset
+def subset(client, source_roots):
+    def file2test(f: str):
+        if f.endswith('.java'):
```
Comment (ninjinkun) [inline launchable/test_runners/maven.py:10]: Do you think we need to support `.kt` and `.scala` like our Gradle plugin, don't you? IIRC, Maven also can handle Scala and Kotlin.

Comment (Konboi) [reply launchable/test_runners/maven.py:10]: You're right. I hadn't considered that.
the maven is described in the [kotilin official documentation](https://kotlinlang.org/docs/reference/using-maven.html), so I'll fix the same with the gradle plugin

Comment (Konboi) [reply launchable/test_runners/maven.py]: fixed: https://github.com/launchableinc/cli/pull/118/commits/240905b6a645bb23bf26d4240be26f3a99513041

**Review by ninjinkun — APPROVED** [review]
LGTM

**Pull request merged**